### ISSUE TITLE
Constant Current not working in 1.0.3

### ISSUE THREAD

**Issue by TobiasPSP** [adafruit/Adafruit_AW9523#9]
I strictly followed your example code, and while the first two examples (blink and button LED) work as expected, the "constant current" demo fails terribly. In fact, it burned my LEDs. 

I can see resistor-protected LEDs blink twice, then go dark. LEDs w/o resistors go up in steam and flames.

I tested my results with a number of different hardware from Adafruit and others, and could repro in every case. When using a different library, all works fine, thus the issue is definitely caused by a software glitch in your library, 

Since I am sure your example code must have worked before, I assume your recent update has a regression bug and was not properly tested with "constant current" modes. Could you please look into this?

Malfunctioning "constant current", especially when encouraging users to use LEDs w/o protective resistors, causes painful damage.

**Comment by caternuson** [comment]
Which AW9523 pin(s) were you using? This may be related to #8. For pins < 12, `analogWrite()` was essentially not doing anything.

**Comment by TobiasPSP** [comment]
Thank you for your response. 
I strictly followed the example and was using pin 0, not changing a bit. Will try the approach you mentioned in #8.

To be honest, I am surprised to see a commercial library being updated without apparently testing at minimum the three very simple examples that ship with it. The exact example configuration with the pin 0 is clearly failing. 

I typically take issues like this as a welcomed technical challenge, however in this particular case, the fault actually destroys electronic parts completely unexpectedly. That’s not so much fun.

After crucially following the instructions for the „constant current“ example, two LEDs turned to smoke, one of which was already mounted in a larger project. 

I don‘t know if that could occur when analogWrite() would just do nothing. It appears to me that rather the „constant current“ was not enabled altogether for that pin, and the LEDs got full current from default GPIO out.

**Comment by TobiasPSP** [comment]
I just downloaded the version 1.0.4 which apparently includes the mentioned fix. This has also fixed the constant current issues I was experiencing. I also changed the hardware address to 0x5b because according to the manufacturer, this address ensures that all GPIO pins are non-active by default, thus making sure sensitive (unprotected) LEDs aren't damaged by default active GPIO out pins. Thanks for 1.0.4!
